nvdaHelper/readme.md

```diff
@@ -70,7 +70,7 @@ You should still build on the command line to verify errors.
   - Press next
 -  Specify Debug Configuration Settings
   - Build command line: `scons source`
-  - Preprocessor definitions (/D): `WIN32;_WINDOWS;_USRDLL;NVDAHELPER_EXPORTS;UNICODE;_CRT_SECURE_NO_DEPRECATE;LOGLEVEL=15;_WIN32_WINNT=_WIN32_WINNT_WIN7;NOMINMAX`
+  - Preprocessor definitions (/D): `WIN32;_WINDOWS;_USRDLL;NVDAHELPER_EXPORTS;UNICODE;_CRT_SECURE_NO_DEPRECATE;LOGLEVEL=15;_WIN32_WINNT=_WIN32_WINNT_WINBLUE;NOMINMAX`

```

Comment:
Interesting enough, the readme is updated but archBuild_sconscript seems not.

When updating archBuild_sconscript, also consider updating the linker subsystem, I guess that can become 6.03 now, though it is very hard to track down documentation on this.

Reply:
ah it got missed being git added at some point.

Can you explain what you mean by updating the linker subsystem?

Reply:
Sure. search for `subsystem = "/subsystem:windows,6.02"`

Reply:
No updates seem available according to windows docs

https://learn.microsoft.com/en-us/cpp/build/reference/subsystem-specify-subsystem?view=msvc-170#remarks

Reply:
nevermind

Reply:
fixed both of these issues raised